src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -146,80 +149,64 @@ public function downvoteAction(Comment $comment, UserInterface $user)
      */
     private function addComment(Request $request, string $itemType, $itemId, UserInterface $user)
     {
-        $parentId = $request->request->get('parent');
-        $content = $request->request->get('content');
+        $parentId=$request->request->get('parent');
+        $content=$request->request->get('content');
 
-        $em = $this->getDoctrine()->getManager();
-        $comment = $em->getRepository('SaltSiteBundle:Comment')->addComment($itemType, $itemId, $user, $content, $parentId);
+        $em=$this->getDoctrine()->getManager();
+        $comment=$em->getRepository('SaltSiteBundle:Comment')->addComment($itemType, $itemId, $user, $content, $parentId);
 
         return $this->apiResponse($comment);
     }
 
     private function serialize($data)
     {
         return $this->get('jms_serializer')
-            ->serialize($data, 'json');
+        ->serialize($data, 'json');
     }
 
-    private function apiResponse($data, $statusCode = 200): JsonResponse
+    private function apiResponse($data, $statusCode=200): JsonResponse
     {
-        $json = $this->serialize($data);
+    $json=$this->serialize($data);
 
-        return JsonResponse::fromJsonString($json, $statusCode);
-    }
+    return JsonResponse::fromJsonString($json, $statusCode);
+}
 
-    /**
-     * @Route("/salt/case/export_comment/{itemType}/{id}/comment.csv", name="export_comment_file")
-     *
-     * @param int $id
-     * @param string $itemType
-     * @param Request $request
-     *
-     * @return Response
-     */
-    public function exportCommentAction(string $itemType, int $id, Request $request)
+/**
```

Comment:
Fix wrong indentation in this entire block. Also we should keep private functions at the end of the file.